Controllers/CouriersController.go

```diff
@@ -26,8 +26,22 @@ func (h *Handler) GetOneCourier(c *gin.Context) {
 	}
 	Courier, err = h.services.GetCourier(id)
 	if err != nil {
-		c.JSON(http.StatusBadRequest, gin.H{"No such courier": err})
+		c.JSON(http.StatusInternalServerError, gin.H{"No such courier": err})
 		return
 	}
 	c.JSON(http.StatusOK, Courier)
 }
+
+func (h *Handler) SaveCourier(c *gin.Context) {
```

Comment:
I was advised to name the variable at least ctx instead of c. Variable names can be short, but still should be meaningful. One letter is not an option.

Reply:
fixed